x/payment/keeper/out_flow.go

```diff
@@ -0,0 +1,88 @@
+package keeper
```

Comment:
for the `price.go` file, please avoid using `operate address` of a SP in either key or value in store.

Reply:
refactor the done